addOns/ascanrules/src/test/java/org/zaproxy/zap/extension/ascanrules/TestCrossSiteScriptV2UnitTest.java

```diff
@@ -89,6 +91,49 @@ protected Response serve(IHTTPSession session) {
         assertThat(alertsRaised.get(0).getConfidence(), equalTo(Alert.CONFIDENCE_MEDIUM));
     }
 
+    @Test
+    public void shouldReportXssInParagraphForNullBytePayloadInjection()
```

Comment:
Please include `// Given`, `// When`, and `// Then` comment tags as elsewhere in this test class.
https://github.com/zaproxy/zaproxy/wiki/DevGuidelines#unit-test-specific-guidelines

Reply:
done the changes, please review.